circuit_puzzles/programs/announcer_configure.clsp

```diff
@@ -1,7 +1,7 @@
-(mod ((MOD_HASH STATUTES_STRUCT LAUNCHER_ID INNER_PUZZLE_HASH APPROVED DEPOSIT DELAY ATOM_VALUE COOLDOWN_START TIMESTAMP_EXPIRES)
+(mod ((MOD_HASH STATUTES_STRUCT LAUNCHER_ID INNER_PUZZLE_HASH APPROVED DEPOSIT DELAY ATOM_VALUE COOLDOWN_START LAST_PENALTY_INTERVAL TIMESTAMP_EXPIRES)
       ; solution
       inner_puzzle_hash new_puzzle_hash deposit input_conditions
-      (current_timestamp statutes_inner_puzzle_hash input_toggle_activation deactivation_cooldown_interval delay new_atom_value melt)
+      (current_timestamp statutes_inner_puzzle_hash input_toggle_activation deactivation_cooldown_interval min_deposit delay new_atom_value melt)
```

Comment:
rename delay -> max_delay and have an extra solution arg delay in the row above? this way we can check that delay <= max_delay further down, allowing Announcers to reduce their respective announcer's delay prior to a delay reduction by governance. o/w Announcers would need to scramble to reduce their delay in the same block as the enactment or risk getting penalised straight away.